test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -14,26 +14,42 @@
  * limitations under the License.
  */
 
+@file:Suppress("invisible_reference", "invisible_member")
```

Comment:
I think we should strive to avoid having these at file level? What is the purpose?

I have used utility methods or extension properties elsewhere to encapsulate the internal access, to ensure that we don't leak all internal symbols into the full test.

Reply:
I agree but here I'm using it to access `AggregatorQueryType` and switch over its values so I need the import which cannot be found unless I put the annotation at a file level. Writing the full path to the enum would work too but it's used quite a lot of times so I don't think that scales up.

Reply:
Then I think we should consider making the class public. As long as it is in an `internal` sub-package then the IDE will not autocomplete and automatically import it for the public users.